pkg/watchdogd/watchdogd.go

```diff
@@ -71,73 +100,148 @@ func MonitorOops() error {
 	return nil
 }
 
+type opHandler struct{}
+
+// ServeHTTP processes watchdog operation by integer in URL path and writes back result in either 0 (OK) or 1(ERROR).
```

Comment:
I wonder if you need all the complexity of an http server. You could I think create a unix domain socket in /tmp and relay one-byte requests back and forth. This brings in a lot less apparatus and source of error than a full http stack.

Reply:
not at all, over unix network is lighter, and entirely local, certainly desired here.

our discussion came down to that we need a **ACK** of Done from watchdogd for disarming, one way or other.

updated to use unix socket